src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -296,6 +304,29 @@ private String processGenotypeField(int alternateAlleleIdx, String genotype) {
         return genotype.intern();
     }
 
+    protected void checkForNonVariantSites(Variant variant, String fileId, String studyId) throws VCFRecordWithNonVariantSites {
+
+        //All the sample genotypes are non-variant (./., 0/0, 0|0)
```

Comment:
IMO this comments are a bit redundant. I try to avoid them because they usually get outdated soon. Instead of putting a little effort writing comments, use that effort to make the code more readable.

Reply:
@jmmut Thanks for the comments! I'll fix these issues :)